src/Files.Uwp/ViewModels/FolderSettingsViewModel.cs

```diff
@@ -690,6 +549,7 @@ public void OnDefaultPreferencesChanged(string folderPath, string settingsName)
                     break;
                 case nameof(UserSettingsService.PreferencesSettingsService.AreLayoutPreferencesPerFolder):
                     LayoutPreference = prefs;
+                    // TODO: update layout
```

Comment:
Is this something being removed by the PR or was this never the case? Maybe this is worth tracking via an issue?

Reply:
Ah no this is just adding the comment. But worth opening an issue.